src/app/login/components/Login.tsx

```diff
@@ -69,12 +70,24 @@ export const LoginPage = () => {
   } = form;
 
   return (
+    <FormWrapper>
     <div className="h-full flex items-center justify-center p-4">
-      <div className="bg-white rounded-lg shadow-xl overflow-hidden max-w-md w-full">
-        <div className="bg-gray-800 p-8 text-center">
-          <h2 className="text-2xl font-bold text-white">Welcome Back</h2>
-          <p className="text-gray-400">Log in to your Shoes Rack account</p>
-        </div>
+    <div className="bg-white rounded-lg shadow-2xl overflow-hidden max-w-md w-full">
+      <div className="flex justify-center item-center pt-8 ">
+        <Image
+          src={LogoImage}
+          width="65"
```

Comment:
can we add even number instead of odd number for giving width and height because it is standard to use even numbers

Reply:
@Mubashar1009 i think its much important to adjust image that will looks good.